### PR TITLE
Add funtion returning info on GV control access

### PR DESCRIPTION
**Pull request by casperroo**
arv_gv_device_is_controller returns value indicating whether the ArvGvDevice has control access to the camera

Currently the 0.8.0 API does not provide any function that tells whether the user has a control access to the camera (GV).
The ArvGvDeviceIOData has this information as the initialization process attempts to gain control over the camera by means of arv_gv_device_take_control function. If it fails the function yields debug warning but this is not reported to the user otherwise.
Subsequent GeniCam writes fail obviously but report timeout as the reason. So if the camera was initialized without control access, the arv_gv_device_heartbeat_thread will never emit control_lost_signal.

I propose to add a function that simply returns the private field gv_device->priv->io_data->is_controller.
I am new to this project so I do not know if this is right. This method may require mutex as arv_gv_device_heartbeat_thread may update this field.

I have tested the above on Lucid and Basler cameras.

### CODE REVIEW

File: src/arvgvdevice.c [hunk 1]
```diff
@@ -672,6 +672,25 @@ arv_gv_device_auto_packet_size (ArvGvDevice *gv_device, GError **error)
...
+ *
+ * Since: 0.8.0
+ */
+
+gboolean
+arv_gv_device_is_controller (ArvGvDevice *gv_device)
+{
+	g_return_val_if_fail (ARV_IS_GV_DEVICE (gv_device), 0);
+	
+	ArvGvDevicePrivate *priv = arv_gv_device_get_instance_private (gv_device);
```
Comment (EmmanuelP) [inline src/arvgvdevice.c:689]: Variable declaration should come at the beginning of a code block. It is safe to move this line at the beginning of the bloc, as it doesn't actually dereference gv_device.

File: src/arvfeatures.h.in [hunk 2]
```diff
@@ -47,4 +47,12 @@
 
 #define ARAVIS_HAS_PACKET_SOCKET @ARAVIS_HAS_PACKET_SOCKET@
 
+
+#define ARAVIS_HAS_FAST_HEARTBEAT @ARAVIS_HAS_FAST_HEARTBEAT@
```
Comment (EmmanuelP) [inline src/arvfeatures.h.in:51]: This code change is not related to the pull request

Comment (casperroo) [reply src/arvfeatures.h.in]: Sorry - I have committed to the same branch changes related to two different pull requests

**Comment by EmmanuelP** [comment]
Thanks.
